Author: Kaz Kylheku <kkylheku@sierrawireless.com>
Date:   Wed Mar 15 17:55:32 2017 -0700

This removes a declaration/definition inconsistency which
breaks the build. The attribute added here on the declaration
creates some sort of the clash against the function definition.

diff --git a/gcc/cp/cfns.h b/gcc/cp/cfns.h
index c845ddf..705fd33 100644
--- a/gcc/cp/cfns.h
+++ b/gcc/cp/cfns.h
@@ -124,9 +124,6 @@ hash (register const char *str, register unsigned int len)
 
 #ifdef __GNUC__
 __inline
-#ifdef __GNUC_STDC_INLINE__
-__attribute__ ((__gnu_inline__))
-#endif
 #endif
 const char *
 libc_name_p (register const char *str, register unsigned int len)
